GsaGHTests/Helpers/Assemble/AssembleModelLists.cs

```diff
@@ -0,0 +1,56 @@
+﻿using GsaGH.Components;
+using GsaGH.Parameters;
+
+using Xunit;
+
+namespace GsaGHTests.Helpers.Assemble {
+  [Collection("GrasshopperFixture collection")]
+  public class AssembleModelLists {
+    private static readonly int DefaultId = 1;
+    private static readonly string DefaultName = "create list test";
+    private static readonly string DefaultDefinition = "1 2 3";
+
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false)]
+    public void ListShouldReturnDefinitionWhenIdIsSet(bool withId) {
+      GsaList gsaList = GsaList(withId);
+
+      Assert.Equal(DefaultId, gsaList.Id);
```

Comment:
I think this Default ID is a bit confusing. If we create a list with specific ID, we should expect to get that ID back. You are using 1, which is also the default ID, therefore conflating them. I would suggest focusing the tests only on a specific non default ID. Also to separate them rather than passing a boolean as it is a bit tricky to follow the logic.

Reply:
will rename variable.
about splitting them - the only difference is that boolean. Don't see the need of separate tests.